imagedephi/redact.py

```diff
@@ -114,32 +114,43 @@ def get_base_rules():
         return base_rule_set
 
 
+def iter_image_files(directory: Path) -> Generator[Path, None, None]:
+    """
+    Given a directory return an iterable of available images.
+
+    May raise a PermissionError if the directory is not readable.
+    """
+    for child in directory.iterdir():
+        if child.suffix == ".tif" or child.suffix == ".svs":
```

Comment:
We should check if it is in `FILE_EXTENSION_MAP`.

Reply:
Are there additional extensions I should include in the map or just add as we go?

Reply:
For now, we have a very limited tool that only works on tiff files and variations.  There are lots of extensions that we've seen for tiff files (ptif for Philips' generated tiff files, for instance).  From a UI perspective, the existing file extension map is file.  Eventually, we'll want to be able to let the user filter by their own method (including trying all files), and not care about file extension.

The more correct approach is to use something like the "magic" library to detect file format, or, more certain, use large_image and its tile sources -- but this requires reading some of each file and is therefore slow, especially in cases where we have thousands of files in a directory.

Reply:
filed as issue #61 to fix after merge if #55  is not merged first